src/axolotl/integrations/nvfp4/kernels/lm_head_ce.py

```
@@ -0,0 +1,95 @@
+from axolotl.utils.logging import get_logger
+
+LOG = get_logger(__name__)
+
+
+def _output_head(model: nn.Module):
+    """Resolve the ForCausalLM output embedding, unwrapping a PEFT wrapper."""
+    causal = model
+    if hasattr(model, "get_base_model"):
+        try:
+            causal = model.get_base_model()
+        except Exception:  # pylint: disable=broad-except
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)